docs/tianocore_build.md

```diff
@@ -0,0 +1,91 @@
+# Coreboot with tianocore payload on apu2
+=========================================
+
+This document describes how to build coreboot image with tianocore payload for
+PC Engines apu2 platform.
+
+## Building coreboot image
+
+1. Clone the [pce-fw-builder](https://github.com/pcengines/pce-fw-builder)
+2. Pull or [build](https://github.com/pcengines/pce-fw-builder#building-docker-image)
+  docker container:
+
+  ```
+  docker pull pcengines/pce-fw-builder
+  ```
+
+3. Build v4.9.0.6 image:
+
+  ```
+  ./build.sh release v4.9.0.6 apu2
+  ```
+
+4. Invoke distclean:
+
+  ```
+  ./build.sh dev-build $PWD/release/coreboot apu2 distclean
+  ```
+
+5. Copy config file for target platform
+
+  ```
+  cp $PWD/release/coreboot/configs/config.pcengines_apu2 $PWD/release/coreboot/.config
+  ```
+
+6. Create full config:
+
+  ```
+  ./build.sh dev-build $PWD/release/coreboot apu2 olddefconfig
+  ```
+
+7. Invoke menuconfig:
+
+  ```
+  ./build.sh dev-build $PWD/release/coreboot apu2 menuconfig
+  ```
+
+8. In menuconfig go to `Payload` menu and next:
+
+  a. In `Add a payload` choose *Tianocore coreboot payload package*
```

Comment:
@piotr-kleins one more thing. Change the enumerated list from :
a) step 1
b) step 2
c)...
To
- step 1
- step 2
- step 3

Reply:
Done